superset/assets/src/dashboard/stylesheets/filter-indicator-tooltip.less

```diff
@@ -75,4 +76,4 @@
   li {
     margin-bottom: 8px;
   }
-}
+}
```

Comment:
please add a new line at the end of file :)

Reply:
Done! Also added a stylelint rule to help catch this in the future.